### DISCUSSION TITLE
Will the AlPypeRL work with the latest version of anylogic?

### DISCUSSION THREAD

**Discussion by ShravanPagrut** [MarcEscandell/ALPypeRL#12] [Q&A]
Hi, 
Does the AlPype Library work with the latest version of Anylogic? If it doesn't , is there a way to make it work with it that can be done on the user end?

**Comment by MarcEscandell** [comment]
Hi @ShravanPagrut ! Would you mind sharing the exact version you are referring to?

**Comment by ShravanPagrut** [comment]
Hi @MarcEscandell , we wanted to run it on ver 8.9.3.
